src/main/java/com/capstone/doctorsOffice/services/AppointmentService.java

```diff
@@ -0,0 +1,22 @@
+package com.capstone.doctorsOffice.services;
+
+import com.capstone.doctorsOffice.dtos.AppointmentDto;
+import jakarta.transaction.Transactional;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface AppointmentService {
+    @Transactional
+    List<String> addNewAppointment(AppointmentDto appointmentDto);
+
+    @Transactional
+    void deleteAppointment(Long appointmentId);
+
+    @Transactional
+    void updateAppointment(AppointmentDto appointmentDto);
+
```

Comment:
update function signature in interface to take the additional parameter
void updateAppointment(Long  appointment, AppointmentDto appointmentDto)

Reply:
Done